### ISSUE TITLE
Docs are not good enough

### ISSUE THREAD

**Issue by rdbartram** [worxspace/PSPulumi#7]
@Oechiih can you address this please

**Comment by cveld** [comment]
Yes I was looking at your conference talk and the relation between the Pulumi PowerShell implementation towards you guys and Pulumi was a bit unclear to me. https://www.youtube.com/watch?v=YAQLvNjJKOs
Maybe good to add something on this onto a readme.md into this repo.
Also cross referencing this to https://github.com/pulumi/pulumi-azure-native/discussions/1818
